views/base.html.twig

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <!-- Always force latest IE rendering engine (even in intranet) & Chrome Frame
+  Remove this if you use the .htaccess -->
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+
+  <title>{% block title %}Magazine Theme for Superdesk Publisher{% endblock %}</title>
+  <meta name="author" content="Sourcefabric">
+
+  <meta name="description" content="Superdesk Web Publisher is state of the art publishing system.">
+  <meta name="keywords" content="Superdesk, Sourcefabric, open source, media, journalism">
+
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+
+
+
+  <link rel='stylesheet' href='/public/dist/style.css' type='text/css' media='all' />
+  <link rel="preload" href="/public/dist/all.js" as="script">
+
+  <link rel="manifest" href="/manifest.json">
+
+  <link rel="shortcut icon" href="/public/img/icons/favicon.ico">
+  <link rel="apple-touch-icon" sizes="180x180" href="/public/img/icons/apple-touch-icon.png">
+  <link rel="icon" type="image/png" href="/public/img/icons/favicon-32x32.png" sizes="32x32">
+  <link rel="icon" type="image/png" href="/public/img/icons/favicon-16x16.png" sizes="16x16">
+  <link rel="mask-icon" href="/public/img/icons/safari-pinned-tab.svg" color="#4866a4">
+  <meta name="theme-color" content="#4866a4">
+
+  {% block head %}{% endblock %}
+
+
+</head>
+<body>
+  <header class="header">
+    <button class="header__hamburger"></button>
+    <div class="header__logo">
+      <a href="/" class="logo"></a>
+    </div>
+
+
+    {# {% container 'mainNav' with {
+        'cssClass': 'nav header__nav okayNav',
+        'data': {'id': 'nav', 'role': 'navigation'}
+        }
+    %} #}
+
+     {{ knp_menu_render('mainNavigation',{'template':'widgets/mainNav.html.twig'}) }}
+
+    {# {% endcontainer %} #}
+  </header>
+  <div class="overlay"></div>
+  {{ knp_menu_render('mainNavigation',{'template':'widgets/mobileNav.html.twig'}) }}
+
+
+{% block body %}{% endblock %}
+
+
+  <script type="text/javascript">
+    WebFontConfig = {
+      google: { families: [ 'Open+Sans:300,300i,400,400i,600,600i,700,800' ] }
```

Comment:
ask Backo if we need open sans. Removing it won't improve load time that much. It is loaded asynchronously and cached.